lab_3_ann_retriever/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-    "target_score": 0
+    "target_score": 10
```

Comment:
Too much additional typings! But the implrmrntation looks good!

Reply:
Thank you!

Reply:
Again - remove typings inside functions